### ISSUE TITLE
stig-rhel6-disa audit actions not accepted

### ISSUE THREAD

**Issue by jamescassell** [OpenSCAP/scap-security-guide#2755]
#### Description of problem:
the xccdf for audit actions allows multiple actions, such as: `Acceptable values also include syslog, exec, single, and halt.`

#### SCAP Security Guide Version:
0.1.38

#### Operating System Version:
RHEL 6.9

#### Steps to Reproduce:

1. manually remediate system per STIG
2. run oscap
3. see failure

#### Actual Results:
```
Configure auditd admin_space_left Action on Low Disk Space - Fail
Configure auditd space_left on Low Disk Space - Fail
Configure auditd Disk Full Action when Disk Space Is Full - Fail
Configure auditd Disk Error Action on Disk Error - Fail
```

#### Expected Results:
pass

#### Addition Information/Debugging Steps:
similar to #2751 it seems the oval does not accept all permittable values.

**Comment by redhatrises** [comment]
@jamescassell did you tailor the profile at all or no? These options expect that the file be tailored.

**Comment by jamescassell** [comment]
no, I'm just running it out of the box.

**Comment by redhatrises** [comment]
Closing as this is a tailoring profile issue from defaults.

**Comment by shawndwells** [comment]
@redhatrises appears the default content is being used (not tailored). Current SSG content has different acceptable values than what DISA has quietly updated to:
`````
    Set this to <tt>email</tt> (instead of the default,
    which is <tt>suspend</tt>) as it is more likely to get prompt attention. Acceptable values
    also include <tt>suspend</tt>, <tt>single</tt>, and <tt>halt</tt>.
`````

**Comment by shawndwells** [comment]
Also, looks like all the variables were stripped out during the YAML conversion

**Comment by shawndwells** [comment]
$ pwd
/var/www/html/github/openscap/scap-security-guide/shared/guide/system/auditing
$ grep -rin var_auditd_space_left_action
(no results)

**Comment by yuumasato** [comment]
The default value for `var_auditd_space_left_action` is `email`, check the var [file](https://github.com/OpenSCAP/scap-security-guide/blob/master/linux_os/guide/system/auditing/configure_auditd_data_retention/var_auditd_space_left_action.var#L12)
This may have been fixed by#2933.

Please, reopen if not.
